packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx

```diff
@@ -114,6 +125,11 @@ function createSignals(signalsConfig: SignalsV1, validate: boolean): Signals {
       const s = signal.parse(undefined)
```

Comment:
I think this might be where you want to use the initial value (instead of undefined) instead of where you have it a few lines down. Parsing undefined gets the default value but in the case when a value's been passed in we want to call parse on that passed in value